project2/style.css

```diff
@@ -0,0 +1,66 @@
+.title {
+    font-family: 'ZCOOL QingKe HuangYou', cursive;
+    color: white;
+    font-size: 19pt;
+    line-height: 50px;
+    margin-left: 8px;
+    padding-right: 20px;
+
+}
+
+.rate {
+    font-family: 'Khand', sans-serif;
+    font-size: 15pt;
+    display: table;
+    color: white;
+    position: absolute;
+    left: 150px;
+    top: 100px;
+    bottom: 60px;
+}
+
+.context {
+    width: 345px;
+    height: 240px;
+    position: absolute;
+    top: 220px;
+    background-color: silver;
+    opacity: 0.7;
+}
+
+.rapper {
```

Comment:
The word you want to use here (and in the class value in the HTML) is "wrapper" -- as in, this _wraps_ around something. A classname of "card" would also be appropriate for this assignment.

This is just an FYI and has no effect on your grade. 👍🏻